security/advancedtls/crl.go

```diff
@@ -241,23 +242,23 @@ func fetchIssuerCRL(crlDistributionPoint string, rawIssuer []byte, crlVerifyCrt
 
 // checkCrt checks a single certificate against the CRL defined in the certificate.
 // It will fetch and verify the CRL(s) defined by CRLDistributionPoints.
-// If we can't load any authoritative CRL files, the status is UNDETERMINED
+// If we can't load any authoritative CRL files, the status is RevocationUndetermined
 // c is the certificate to check
 // crlVerifyCrt is the group of possible certificates to verify the crl
 func checkCrt(c *x509.Certificate, crlVerifyCrt []*x509.Certificate, cfg RevocationConfig) RevocationStatus {
 	if len(c.CRLDistributionPoints) == 0 {
-		return UNREVOKED
+		return RevocationUnrevoked
 	}
 	// Iterate through CRL distribution points to check for status
 	for _, dp := range c.CRLDistributionPoints {
 		crl, err := fetchIssuerCRL(dp, c.RawIssuer, crlVerifyCrt, cfg)
 		if err != nil {
-			log.Printf("getIssuerCRL(%v) err = %v", c.Issuer, err)
+			grpclog.Printf("getIssuerCRL(%v) err = %v", c.Issuer, err)
```

Comment:
Similar: please review the error conditions and determine whether it should be info/warning/error.  (I believe "print" is the same as "info".)  We have a loosely-defined [guide](https://github.com/grpc/grpc-go/blob/master/Documentation/log_levels.md) to help determine this.